engine/itest/src/test/java/org/opennms/oce/engine/itest/TopologyEngineTest.java

```diff
@@ -68,72 +67,32 @@ public void canRunEngineWithNoAlarms() {
     }
 
     @Test
-    public void canTriggerIncidentOnCardDown() {
+    public void testSimpleCardScenario() {
         final List<Alarm> alarms = new ArrayList<>();
+        // Fail all ports on all cards of node: n1
         alarms.addAll(new MockAlarmBuilder()
                 .withId("a1")
                 .withResourceKey(new ResourceKey("Port,n1-c1-p1"))
                 .withEvent(SECONDS.toMillis(1), Severity.MAJOR)
-                .withEvent(SECONDS.toMillis(301), Severity.CLEARED) // 5 minutes later
                 .build());
         alarms.addAll(new MockAlarmBuilder()
                 .withId("a2")
                 .withResourceKey(new ResourceKey("Port,n1-c1-p2"))
                 .withEvent(SECONDS.toMillis(31), Severity.MAJOR)
-                .withEvent(SECONDS.toMillis(331), Severity.CLEARED) // 5 minutes later
-                .build());
-        alarms.addAll(new MockAlarmBuilder()
-                .withId("a3")
-                .withResourceKey(new ResourceKey("Port,n2-c1-p1"))
-                .withEvent(SECONDS.toMillis(61), Severity.MAJOR)
-                .withEvent(SECONDS.toMillis(121), Severity.CLEARED) // ~1 minute later
                 .build());
 
         List<Incident> incidents = run(alarms);
 
-        assertThat(incidents, hasSize(4));
-        // The 2nd incident is the Card Down and must contain the 2 alarms
-        Incident incident = incidents.get(1);
+        assertThat(incidents, hasSize(1));
+        // Incident is created and contains both causing alarms.
+        Incident incident = incidents.get(0);
+        // FIXME - IncidentRefactor - assertThat(incident.getModelObject().getType(), equalTo("Card"));
+        // FIXME - IncidentRefactor - assertThat(incident.getModelObject().getOperationalState().toString(), equalTo(OperationalState.SA.toString()));
```

Comment:
Perhaps we should use the incident severity to reflect the operational state of the model object and assert on that instead.